app/Routes.jsx

```diff
@@ -12,28 +12,53 @@ import Settings from './containers/Settings'
 import DisplayWalletAccounts from './containers/DisplayWalletAccounts'
 import Home from './containers/Home'
 
-export default () => (
-  <App>
-    <Switch>
-      <Route path={ROUTES.DASHBOARD} component={Dashboard} />
-      <Route
-        path={ROUTES.CREATE_WALLET}
-        render={props => <CreateWallet {...props} encryptWIF={false} />}
-      />
-      <Route
-        path={ROUTES.ENCRYPT_KEY}
-        render={props => <CreateWallet {...props} encryptWIF />}
-      />
-      <Route path={ROUTES.LOGIN_PRIVATE_KEY} component={LoginPrivateKey} />
-      <Route path={ROUTES.LOGIN_LOCAL_STORAGE} component={LoginLocalStorage} />
-      <Route path={ROUTES.LOGIN_NEP2} component={LoginNep2} />
-      <Route path={ROUTES.LOGIN_LEDGER_NANO_S} component={LoginLedgerNanoS} />
-      <Route path={ROUTES.SETTINGS} component={Settings} />
-      <Route
-        path={ROUTES.DISPLAY_WALLET_KEYS}
-        component={DisplayWalletAccounts}
-      />
-      <Route path={ROUTES.HOME} component={Home} />
-    </Switch>
-  </App>
-)
+export default class Routes extends Component<Props> {
+  state = {
+    blockHeight: 0
+  }
+  render () {
+    return (
+      <App {...this.state}>
+        <Switch>
+          <Route
+            path={ROUTES.DASHBOARD}
+            render={() => (
+              <Dashboard
+                handlePassBlockHeight={blockHeight => {
+                  if (blockHeight !== this.state.blockHeight) {
+                    console.log('setting block height')
```

Comment:
sneaky console.log

Reply:
I don't really like this method... `handlePassBlockHeight` feels like a hacky patch for this problem.